watcher/watcher.go

```diff
@@ -286,11 +286,9 @@ func (w *Watcher) reportRoomsStatusesRoutine() {
 		case <-podStateCountTicker.C:
 			w.PodStatesCount()
 		case <-roomStatusTicker.C:
-			w.Logger.Info("Start to report rooms status")
```

Comment:
Why remove these logs?

Reply:
These ones weren't very helpful. But I can leave the logs there too